custom/icds_reports/data_pull/consolidated_monthly_data_pull.sql

```diff
@@ -0,0 +1,115 @@
+sum(awc_days_open),
+CASE WHEN num_launched_awcs>0 THEN awc_days_open/num_launched_awcs ELSE awc_days_open END AS "Avg. # of Days AWCs open",
+sum(num_awc_infra_last_update) AS "# of AWCs that submitted Infra form",
+SUM(cases_household) AS "# Households Registered",
+SUM(cases_ccs_pregnant) AS "# Pregnant Women",
+SUM(cases_ccs_lactating) AS "# Lactating Mothers",
+SUM(cases_person_adolescent_girls_11_14) AS "# Adolescent Girls (11-14y)",
+SUM(cases_child_health) AS "# Children (0-6y)"
+FROM agg_awc_monthly
+WHERE aggregation_level=1 AND month='2019-11-01'
+ORDER BY state_name
+) TO '/tmp/monthly_stats1.csv' DELIMITER ',' CSV HEADER ENCODING 'UTF-8';
+
+
+-- # of AWCs that reported available drinking water
+-- # of AWCs that reported unavailable drinking water
+-- # of AWCs that reported available functional toilet
+-- # of AWCs that reported unavailable functional toilet
+-- # of AWCs that reported usable infantometer
+-- # of AWCs that reported unavailable usable infantometer
+-- # of AWCs that reported usable stadiometer
+-- # of AWCs that reported unavailable usable stadiometer
+-- # of AWCs that reported available medicine kit
+-- # of AWCs that reported unavailable medicine kit
+-- # of AWCs that reported available infant weighing scale
+-- # of AWCs that reported unavailable infant weighing scale
+-- # of AWCs that reported available mother and child weighing scale
+-- # of AWCs that reported unavailable mother and child weighing scale
+COPY(
+SELECT state_name,
+count(*) FILTER (WHERE infra_clean_water=1) AS "# of AWCs that reported available drinking water",
+count(*) FILTER (WHERE infra_clean_water=0) AS "# of AWCs that reported unavailable drinking water",
+count(*) FILTER (WHERE infra_functional_toilet=1) AS "# of AWCs that reported available functional toilet",
+count(*) FILTER (WHERE infra_functional_toilet=0) AS "# of AWCs that reported unavailable functional toilet",
+count(*) FILTER (WHERE infantometer=1) AS "# of AWCs that reported usable infantometer",
+count(*) FILTER (WHERE infantometer=0) AS "# of AWCs that reported unavailable usable infantometer",
+count(*) FILTER (WHERE stadiometer=1) AS "# of AWCs that reported usable stadiometer",
+count(*) FILTER (WHERE stadiometer=0) AS "# of AWCs that reported unavailable usable stadiometer",
+count(*) FILTER (WHERE infra_medicine_kits=1) AS "# of AWCs that reported available medicine kit",
+count(*) FILTER (WHERE infra_medicine_kits=0) AS "# of AWCs that reported unavailable medicine kit",
+count(*) FILTER (WHERE infra_infant_weighing_scale=1) AS "# of AWCs that reported available infant weighing scale",
+count(*) FILTER (WHERE infra_infant_weighing_scale=0) AS "# of AWCs that reported unavailable infant weighing scale",
+count(*) FILTER (WHERE infra_adult_weighing_scale=1) AS "# of AWCs that reported available mother and child weighing scale",
+count(*) FILTER (WHERE infra_adult_weighing_scale=0) AS "# of AWCs that reported unavailable mother and child weighing scale",
```

Comment:
This will also include those awcs who have not answered the question along with those who have answered no in that question as its defaulted to 0  `GREATEST(adult_scale_available, adult_scale_usable, 0) AS infra_adult_weighing_scale,`. same for infant_scale Is that acceptable?